### PR TITLE
Run tests using docker

### PR DESCRIPTION
**Pull request by codeliner**
Added a docker-compose config to spin up postgres and mysql containers and link them to a `prooph/composer:71` container.

Added new composer scripts and instructions how to run them, see updated README

Changed `TestUtil` and phpunit config to use env vars so that the DB_HOST can be changed when running the the tests insight the prooph composer container.

### CODE REVIEW

File: docker-compose-tests.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - POSTGRES_DB=event_store_tests
+
+  mysql:
+    image: mysql
+    ports:
+     - 3306:3306
+    environment:
+      - MYSQL_ROOT_PASSWORD=
+      - MYSQL_ALLOW_EMPTY_PASSWORD=yes
+      - MYSQL_DATABASE=event_store_tests
```
Comment (prolic) [inline docker-compose-tests.yml]: New line at end of file

**Comment by prolic** [comment]
Travis failed

**Comment by codeliner** [comment]
yeah, with a strange error ... I'll correct new line and try again ;)

**Comment by codeliner** [comment]
yeah green :tada:

**Pull request merged**